.pipelines/templates/PackageBuild.yml

```diff
@@ -160,6 +160,12 @@ steps:
         displayName: "Populate RPMs cache"
 
   - script: |
+      toolchain_archive="$(find "$(Pipeline.Workspace)" -name "${{ parameters.customToolchainTarballName }}" -print -quit)"
```

Comment:
What if the toolchain archive is outside of the workspace, but the provided TOOLCHAIN_ARCHIVE still points to it?

Reply:
We use the same logic above in "Populate toolchain" (line 128). I'm assuming that one requirement for this template is a valid toolchain, either built or provided via artifact. The `TOOLCHAIN_ARCHIVE` value itself doesn't care where the file is, it just reads the .tar.gz and pulls the files it needs from it.

Reply:
The workspace is the download location for the pipeline artifact and the `customToolchainTarballName` parameter is only telling the template what is the name of the toolchain tarball in that artifact. In other words: the template is hard-coded to expect the toolchain to be passed to it through a pipeline artifact and the build will fail of it's not there.